schemas/com.sendgrid/bounce/jsonschema/2-0-1

```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a SendGrid bounce event. Property descriptions derived from the SendGrid documentation: https://sendgrid.com/docs/for-developers/tracking-events/event/",
+  "self": {
+    "vendor": "com.sendgrid",
+    "name": "bounce",
+    "version": "2-0-1",
+    "format": "jsonschema"
+  },
+  "type": "object",
+  "properties": {
+    "ip": {
+      "description": "The IP address used to send the email. For open and click events, it is the IP address of the recipient who engaged with the email.",
+      "type": "string"
+    },
+    "timestamp": {
+      "description": "The timestamp of when the message was sent",
+      "type": "string",
+      "format": "date-time"
+    },
+    "email": {
+      "description": "The email address of the recipient",
```

Comment:
Having removed the `email` format, could we add a regular expression to make sure that the emails follow some expected format?

```suggestion
      "pattern": "^\\S+@\\S+\\.\\S+$",
      "description": "The email address of the recipient",
```

Reply:
Or is Sendgrid even breaking that expectation and sends invalid email adresses?